1-exercises/C-while-loop-with-array/exercise.js

```diff
@@ -17,7 +17,14 @@ const BIRTHDAYS = [
 ];
 
 function findFirstJulyBDay(birthdays) {
-    // TODO
+  // TODO
+  let capture = "";
+  for (let i = 0; i < birthdays.length; i += 6) {
+    const index = birthdays[i];
+    index;
+    capture += index;
+    console.log(index);
+  }
 }
```

Comment:
Not quite right. Please have a second look.

Hint: Try to use a return value in the function